Add Bert-Large PyTorch Models with various batch sizes

mariecwhite opened the pull request:
Bert-Large is part of our progression suite that we will be monitoring closely at various batch sizes.

Adds ~1min to the CUDA benchmarks. x86 CPU unaffected.

pzread commented on runtime/src/iree/builtins/ukernel/arch/x86_64/mmt4d_x86_64_avx2_fma.c:
```diff
@@ -39,23 +39,15 @@ void iree_uk_mmt4d_tile_f32f32f32_8x8x1_x86_64_avx2_fma(
+    acc4 = _mm256_fmadd_ps(_mm256_broadcast_ss(lhs_ptr + 4), rhs, acc4);
+    acc5 = _mm256_fmadd_ps(_mm256_broadcast_ss(lhs_ptr + 5), rhs, acc5);
+    acc6 = _mm256_fmadd_ps(_mm256_broadcast_ss(lhs_ptr + 6), rhs, acc6);
+    acc7 = _mm256_fmadd_ps(_mm256_broadcast_ss(lhs_ptr + 7), rhs, acc7);
```
Why is this file changed?

mariecwhite replied:
Rebasing error. It's been removed.

pzread commented on build_tools/python/e2e_test_framework/unique_ids.py:
```diff
@@ -85,6 +85,15 @@ def hash_composite_id(keys: Sequence[str]) -> str:
 #    PyTorch.
 MODEL_CLIP_TEXT_SEQLEN64_FP32_TORCH = "9a9515c7-cb68-4c34-b1d2-0e8c0a3620b8"
 MODEL_UNET_2D_FP32_TORCH = "340553d1-e6fe-41b6-b2c7-687c74ccec56"
+MODEL_BERT_LARGE_1x384_FP32_TORCH = "cbc5e400-7c93-4844-aca8-bce8f1bf9948-batch-1"
```
nit: "x" in the middle should be capitalized?

mariecwhite replied:
Updated to "X"

mariecwhite committed:
Add EfficientNet-B7 PyTorch Model (#12768)

Add EfficientNet-B7 model to have better representation of large CNNs
commonly used on servers. At the moment, CNNs in the benchmarking suite
consist of MobileNets (mobile-centric) and ResNets (outdated).

Has minimal (<1 min) effect on x86 and CUDA benchmark time.
